tools/CustomTasks/BatchMergeXaml.cs

```diff
@@ -131,6 +136,17 @@ public override bool Execute()
                 ExecuteForTaskItems(RS5Pages, "RS5");
                 ExecuteForTaskItems(N19H1Pages, "19H1");
             }
+
+            var filesRead = new List<string>();
```

Comment:
Should we also populate a "write" file too?

Reply:
We probably should. It's not needed to fix the current issue, but maybe there is some other underlying issue?

Reply:
Adding the "write" files helps with the 'Clean' scenario, since it knows to delete old generated files. 

I think I even inadvertently hit the scenario where this is important: after doing a merge the FeatureArea.props got reverted, so a different set of xaml/cpp was being compiled. But older xaml files were not getting removed, so I hit build errors.